### DISCUSSION TITLE
Which album does a photo belong to?

### DISCUSSION THREAD

**Discussion by bmaster001** [photoprism/photoprism#3458] [Q&A] [answered]
Maybe a quick and simple question: when you find a photo in the search-view, is there a way to find out which album this photo belongs to (if it's in an album of course)?

**Comment by graciousgrey** [comment, marked as answer]
Not yet, but improvements are planned:

- https://github.com/photoprism/photoprism/issues/373
- https://github.com/photoprism/photoprism/issues/1164

**Reply by bmaster001** [reply]
Thanks!
